client/src/components/AboutPage.js

```diff
@@ -1,11 +1,186 @@
-import React from "react";
-import Page from "./Page";
-import { Container, Row, Col } from "reactstrap";
-import { img } from "./Speaker2.css";
-import hoop from "../images/SVG/hoop.svg";
-import x from "../images/SVG/x.svg";
-import { image } from "../images";
-import ReadMore from './ReadMore'
+import React from 'react';
+import styled from 'styled-components';
+
+import Page from './Page';
+import ReadMore from './ReadMore';
+
+import x from '../images/SVG/x.svg';
+
+import mediaQueryMin from '../styles/MediaQueriesMixin';
+import {
+  AlignCenterColumn,
+  HeadingAligner,
+  HeadingHoop,
+  PageHeading,
+  BreakLineInverted,
+  Heading4,
+  Paragraph,
+  SimpleLink,
+  Heading2,
+  BreakLineMain,
+} from './GlobalStyledComponents/ReversimStyledComps';
+
+// styled-componets components
+
+const IntroContainer = styled.div`
+  ${({ theme: { color, font, space, mq } }) => `
+    padding: calc(2 * ${space.xxl}) 0 calc(3 * ${space.xl}) 0;
+
+    background-color: ${color.background_2};
+    background-image: url(${x});
+    background-repeat: no-repeat;
+    background-size: 600px;
+    background-position: 50% 120%;
+
+    color:  ${color.text_1};
+    font-size: ${font.size_md};
+  `}
+`;
+
+const IntroInnerContainer = styled(AlignCenterColumn)`
+  display: block;
+`;
+
+const MainContainer = styled(AlignCenterColumn)`
```

Comment:
naming; `Main` confuses me, since you actually refer to the body of content, and main is usually used as the mother of all wrappers. so maybe `ContentContainer`